test/gtest/uct/test_flush.cc

```diff
@@ -518,7 +524,7 @@ UCS_TEST_P(uct_flush_test, am_pending_flush_nb) {
         m_flush_flags |= UCT_FLUSH_FLAG_CANCEL;
     }
 
-    test_flush_am_pending(&uct_flush_test::flush_ep_nb, false);
+    test_flush_am_pending(&uct_flush_test::flush_ep_nb, true);
```

Comment:
why are we always destroying ep now?
IMHO we should pass 'true' here only if we use CANCEL flag